packages/mgt-components/src/components/mgt-file-list/mgt-file-upload/mgt-file-upload.scss

```diff
@@ -0,0 +1,92 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+@import '../../../../../../node_modules/office-ui-fabric-core/dist/sass/References';
+@import '../../../styles/shared-sass-variables.scss';
+
+:host .file-upload-button {
+  width: 100%;
+
+  div {
+    float: var(--file-upload-button-float, right);
+    background-color: var(--file-upload-button-background-color, rgb(243, 242, 241));
+    width: 120px;
+    height: 32px;
+    text-align: center;
+    display: table;
+    margin-top: 39px;
+    margin-right: 16px;
+    margin-left: 16px;
+    cursor: pointer;
+    font-size: 14px;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
There is a $font-family var in shared-sass-variable.scss that can be utilized to have font consistency.